config/sync/field.field.paragraph.os2loop_section_page_info_block.os2loop_section_page_attribute.yml

```diff
@@ -0,0 +1,21 @@
+uuid: 64f621d0-829d-42e6-b3ce-042639ae41f4
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.os2loop_section_page_attribute
+    - paragraphs.paragraphs_type.os2loop_section_page_info_block
+  module:
+    - options
+id: paragraph.os2loop_section_page_info_block.os2loop_section_page_attribute
+field_name: os2loop_section_page_attribute
+entity_type: paragraph
+bundle: os2loop_section_page_info_block
+label: Attributes
```

Comment:
I think we should rename the field to `os2loop_section_page_attribute` – or at least change the label to “Type” (which I think is a better name for what it is).

Reply:
Changed label